dapps/marketplace/src/pages/settings/Settings.js

```diff
@@ -304,6 +307,72 @@ class Settings extends Component {
                           />
                         </div>
                       </div>
+                      <div className="form-group row d-flex">
+                        <div className="col-sm">
+                          <label htmlFor="indexing">
+                            <fbt desc="settings.proxyAccountsLabel">
```

Comment:
Please run npm translate before landing so that those new strings get picked up by crowdin.